masterlist.yaml

```diff
@@ -8576,11 +8598,11 @@ plugins:
       - crc: 0xC15BAADF
         util: 'SSEEdit v4.0.3'
   - name: 'BetterQuestObjectives-EFFPatch.esp'
-    req: [ 'EFFDialogue.esp' ]
```

Comment:
That is required as well, since they can group `EFFDialogue.esp` after the patch.

![image](https://user-images.githubusercontent.com/1944639/91641220-cc8c0880-ea1a-11ea-82c5-eb43f68ea11d.png)